### PR TITLE
Update enable_personalization.md

### PR DESCRIPTION
**Pull request by todomagichere**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      | master, 4+
| Edition       | 

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/personalization/enable_personalization.md [hunk 1]
```diff
@@ -74,28 +74,47 @@ of the site, provide the credentials that correspond to each of the sites.
...
+                            description: 'sub_title'
+                        456: 
+                            title: 'title'
+                            image: 'image_legend'
+                            description: 'sub_title'   
+                <site_access_name_2>:
+                    site_name: '<site_name_2>' # For example 'FRA store'
+                    host_uri: '%env(FRA_HOST_URI)%'
+                    authentication:
+                        customer_id: '%env(FRA_CUSTOMER_ID)%'
```
Comment (ciastektk) [inline docs/personalization/enable_personalization.md:101]:
```suggestion
                        customer_id: '%env(int:FRA_CUSTOMER_ID)%'
```

File: docs/personalization/enable_personalization.md [hunk 2]
```diff
@@ -74,28 +74,47 @@ of the site, provide the credentials that correspond to each of the sites.
...
+                    included_item_types: [product, article]
+                    output_type_attributes:
+                        123: # content type ID
+                            title: 'title'
+                            image: 'image_legend'
+                            description: 'sub_title'
+                        456: 
+                            title: 'title'
+                            image: 'image_legend'
+                            description: 'sub_title'
```
Comment (ciastektk) [inline docs/personalization/enable_personalization.md:96]:
```suggestion
                            title: 'name'
                            image: 'image'
                            description: 'description'   
```

File: docs/personalization/enable_personalization.md [hunk 3]
```diff
@@ -74,28 +74,47 @@ of the site, provide the credentials that correspond to each of the sites.
...
+                        included_item_types: [product, article]
+                        output_type_attributes:
+                            123: # content type ID
+                                title: 'title'
+                                image: 'image_legend'
+                                description: 'sub_title'
+                            456: 
+                                title: 'title'
+                                image: 'image_legend'
+                                description: 'sub_title'
```
Comment (ciastektk) [inline docs/personalization/enable_personalization.md:117]:
```suggestion
                                title: 'short_title'
                                image: 'primary_image'
                                description: 'sub_title'
```

**Comment by ciastektk** [comment]
@todomagichere, thank you for your contribution! Please add requested changes, and we can merge this PR.

**Comment by todomagichere** [comment]
@ciastektk it's all ok?

File: docs/personalization/enable_personalization.md [hunk 4]
```diff
@@ -231,6 +243,8 @@ you must run the export separately for each of the `<site_access_name>`/`<custom
 
 ``` bash
 php bin/console ibexa:personalization:run-export
+    --customer-id=<CUSTOMER_ID>
+    --license-key=<LICENSE_KEY>
```
Comment (ciastektk) [inline docs/personalization/enable_personalization.md:247]: These options are duplicated

```suggestion
    --customer-id=<CUSTOMER_ID>
    --license-key=<LICENSE_KEY>
```

**Review by ciastektk — CHANGES_REQUESTED** [review]
Everything is ok except duplicate options

File: docs/personalization/enable_personalization.md (hunk 4 above)
Comment (todomagichere) [reply docs/personalization/enable_personalization.md]: Dah, done right now :)

**Review by ciastektk — APPROVED** [review]

**Review by juskora — APPROVED** [review]

**Pull request merged**